tests/unit/acquisition/test_rule.py

```diff
@@ -348,7 +348,7 @@ def test_joint_batch_acquisition_rule_acquire(
         _, query_point = points_or_stateful(None)
     else:
         query_point = points_or_stateful
-    npt.assert_allclose(query_point, [[0.0, 0.0]] * num_query_points, atol=1e-3)
+    npt.assert_allclose(tf.constant(query_point), [[0.0, 0.0]] * num_query_points, atol=1e-3)
```

Comment:
is this bcs `query_point` can be a numpy array?

Reply:
I think it's because numpy typing is now better than tensorflow, so it's not sure that it's ok to pass query points into an npt function. (The stateful acquisition functions may also be confusing it.)